src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
@rahulrkr08 Thanks for elaborating. The reason I brought up `.bx--breadcrumb-item:last-child::after` is that [`:last-child` pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) allows us to remove trailing slash automatically without forcing applications using Carbon breadcrumb see where the last child in the HTML and add write extra value in `class` attribute there.

Having to add extra value in `class` attribute in the last child becomes non-trivial especially when application's JavaScript code constructs breadcrumb items from data - `for()` loop, `.map()`, `ng-repeat`… would have to determine if the item being rendered is the last item or not and set/unset a class name. I still have trouble seeing any particular scenario that justifies added complexity to the HTML structure and/or the repeated DOM creation code.